server/package.json

```diff
@@ -6,7 +6,7 @@
   "dependencies": {
     "@aws-sdk/client-iot": "^3.360.0",
     "@aws-sdk/client-iot-wireless": "^3.360.0",
-    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#v10.2.0",
+    "brave-alert-lib": "https://github.com/bravetechnologycoop/brave-alert-lib#ad-feature/Spanish_translation",
```

Comment:
Just adding this as a reminder to update this line to use the real version number once https://github.com/bravetechnologycoop/brave-alert-lib/pull/58/files is merged and "deployed".